twine-upload.sh

```diff
@@ -133,6 +134,10 @@ if [[ ${INPUT_PRINT_HASH,,} != "false" || ${INPUT_VERBOSE,,} != "false" ]] ; the
     python /app/print-hash.py ${INPUT_PACKAGES_DIR%%/}
 fi
 
+if [[ ${INPUT_DISABLE_PROGRESS_BAR,,} != "false" ]]; then
+    TWINE_EXTRA_ARGS="--disable-progress-bar $TWINE_EXTRA_ARGS"
```

Comment:
I think we can just pass this unconditionally and not have people bother with setting the flag.